minecraft_servers/craftergang/manifest.json

```diff
@@ -2,6 +2,10 @@
   "server_name": "craftergang",
   "nice_name": "Craftergang",
   "direct_ip": "craftergang.de",
+  "server_wildcards": [
+    "%.craftergang.net",
+    "%.craftergang.de"

```

Comment:
Doesn't look like a real wildcard:
<img width="854" alt="image" src="https://github.com/user-attachments/assets/98c1049b-7e99-45b2-a852-27b632fa56d6">

Reply:
Craftergang is online. again.
![image](https://github.com/user-attachments/assets/be281a6c-1d1b-4268-b395-dbdfd8952fa9)
![image](https://github.com/user-attachments/assets/768773d2-b499-4ffb-8473-d82a7cdb326a)

Reply:
An entry in the `server_wildcards` arrays means that users can join the server with any subdomain. That's why I made a DNS query to a random subdomain above and saw that it doesn't point to the server at all. Maybe this will help you: https://en.wikipedia.org/wiki/Wildcard_DNS_record
This is btw the same as here previously:
- https://github.com/LabyMod/server-media/pull/1049#discussion_r1553532949
- https://github.com/LabyMod/server-media/pull/1230#discussion_r1703163008
- https://github.com/LabyMod/server-media/pull/1230#discussion_r1711419918